modules/biofuels/README.md

```diff
@@ -0,0 +1,49 @@
+# Easy Energy Modules - biofuels
+
+A module preparing biofuels cost and potentials
+
+## Input-Ouput
+
+Here is a brief IO diagram of the module's operation.
+
+```mermaid
+---
+title: biofuels
+---
+flowchart LR
+    D1[("`**automatic**
+        biomass potentials (JRC ENSPRESO)
+        population (JRC GHS-POP)
+        land cover (https://doi.org/10.5281/zenodo.3244985)
+        ...
+    `")] --> |Download| M
+    C1[/"`**user**
+        spatial_units.geojson
+        ...
+    `"/] --> M((biofuels))
+    M --> O1("
+        results/{resolution}/{scenario}/potential-mwh-per-year.csv
+        ")
+    M --> O2("
+        results/{resolution}/{scenario}/costs-eur-per-mwh.csv
+        ")
+```
```

Comment:
Small note: the mermaid diagram does not look nice in firefox, but works on chrome.
This is a known bug, and we can't really do much until mermaid fixes it.